app/cafe.py

```diff
@@ -0,0 +1,19 @@
+import datetime
+
+from app.errors import (OutdatedVaccineError,
+                        NotWearingMaskError,
+                        NotVaccinatedError)
+
+
+class Cafe:
+    def __init__(self, cafe_name: str) -> None:
+        self.name = cafe_name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("Visitor is not vaccinated.")
+        elif visitor["vaccine"]["expiration_date"] < datetime.date.today():
```

Comment:
you need to check all 3 conditions all the time thats why it must be `if` not `elif`